index.css

```diff
@@ -64,97 +84,91 @@ div.base-frame {
   font-family: 'Jua', sans-serif;
 }
 
-.logo {
-  grid-column: 1;
-  align-self: center;
+.page-seperator {
+  background-color: rgb(120, 37, 2);
+  height: 15px;
 }
 
-.logo img {
-  width: 250px;
-  padding-left: 32px;
-}
-
-.image-section {
-  grid-column: 1 / span 3;
-  height: 450px;
-  background-image: 
-    linear-gradient(90deg, rgba(0,0,0,1) 0%, rgba(0,0,0,1) 33%, rgba(245,255,0,0.5) 100%),
-    url('background.jpg');
-  background-blend-mode: overlay;
-  background-position: left;
-  background-size: cover;
-  padding: 0px;
-  margin: 0px;
+.middle-content {
+  background-color: rgb(156, 195, 200);
+  min-height: 100vh;
 }
 
-.image-cta {
-  margin: 0px 50px;
-  padding: 64px;
-  width: 25%;
-  background-color: white;
-  border-radius: 8px;
-  font-family: sans-serif;
+.middle-content-flex-container {
```

Comment:
CSS classes should have somewhat abstract names.  We don't want the implementation of the class to leak into its name (what if the class is changed someday to no longer use flex?). 

Beyond changing the name, a CSS combinator might make more sense here anyway.